src/components/Dashboard.tsx

```diff
@@ -21,5 +25,35 @@ export const Dashboard: React.SFC = () => (
       </div>
     </div>
     <Transactions />
-  </div>
+  </React.Fragment>
 )
+
+export class Dashboard extends React.PureComponent<Props> {
```

Comment:
I'd change this and call it `AccountPage` now.

Reply:
done